### PR TITLE
publish-seven-things-application

### PR DESCRIPTION
**Pull request by emilytraff**
Moving a blog post over from Medium.

### CODE REVIEW

File: app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+Why are you passionate about joining the teaching profession? Come up with your top three reasons why this is the career for you.
+
+## Resilience
+
+What have you found challenging and how did you deal with it? Find an example of a challenge you’ve overcome, and explain what you learnt and how you can apply this to future challenges.
+
+Have confidence in yourself and remember that training providers are on your side! We always enjoy reading about your experiences from school, family, jobs and volunteering, and look forward to meeting you at interview where you can demonstrate further how well-informed and interested you are in becoming a TEACHER.
+
+> Whatever stage you’re at with your application, our expert advisers can help. [Sign up for a Teacher Training Adviser](https://adviser-getintoteaching.education.gov.uk/) who can give you free, one-to-one support.
```
Comment (peteryates) [inline app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md:62]:
```suggestion
Whatever stage you’re at with your application, our expert advisers can help. [Sign up for a Teacher Training Adviser](https://adviser-getintoteaching.education.gov.uk/) who can give you free, one-to-one support.
```

@emilytraff - now the last paragraph will always be styled as a summary so this doesn't need to be quoted. PR is spot on other than that though 👍🏽

**Comment by emilytraff** [comment]
I think this one is good to go?

**Comment by peteryates** [comment]
It looks great @emilytraff - will get it released today

**Comment by GayeCord** [comment]
@EllieNodder could you give this a look over please

@emilytraff just to flag, we have a code freeze due to Christmas. Unless super urgent, no changes will be merged now until 4th Jan at the earliest

File: app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  - becoming a teacher
+  - teacher training
+  - personal statements
+---
+
+$bedfordshire$
+
+_Not sure what to include in your teacher training application? Just think of the word TEACHER! Juliet Fern, Mieka Harris and Kate Hudson-Glynn from the University of Bedfordshire explain all._
+
+So you want to be a teacher — why is this? Were you taught by someone amazing? Do you love a specific subject? Do you want to give something back? Do you like working with others? Do you like to learn new things?
```
Comment (EllieNodder) [inline app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md:26]:
```suggestion
So you want to be a teacher — why? Were you taught by someone amazing? Do you love a specific subject? Do you want to give something back?
```
Think just sticking with three examples reads a bit better

File: app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Many people who join the teaching profession want to inspire the next generation in some way. However getting this across in your application and interview can be tricky. Remember that teacher training providers are looking for your potential to be a teacher, not just what you have already done. It is also really important to get across a feeling of positive energy about your goals and aspirations. Although different training providers will adopt different approaches to the interview process, all are looking for a self-motivated, passionate individual.
```
Comment (EllieNodder) [inline app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md:28]:
```suggestion
Many people who join the teaching profession want to inspire the next generation in some way. However, getting this across in your application and interview can be tricky. Remember that teacher training providers are looking for your potential to be a teacher, not just what you have already done. It is also really important to get across a feeling of positive energy about your goals and aspirations. Although different training providers will adopt different approaches to the interview process, all are looking for a self-motivated, passionate individual.
```

File: app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+What reflections do you have on your experiences of learning and of supporting others to learn? Think about your own learning in and out of school, how you have helped others to improve at something and whether you could have done anything differently.
+
+## Experience
+
+Why do you want to become a teacher? Consider the experiences that have inspired you to apply for teacher training.
+
+## Aptitude
+
+What skills do you have that will make you a good teacher? Think about the qualities your favourite teachers have and research the classroom practice of being a great teacher.
```
Comment (EllieNodder) [inline app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md:42]:
```suggestion
What skills do you have that will make you a good teacher? Think about the qualities your favourite teachers had and research the classroom practice of being a great teacher.
```

File: app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## Enthusiasm
+
+Why are you passionate about joining the teaching profession? Come up with your top three reasons why this is the career for you.
+
+## Resilience
+
+What have you found challenging and how did you deal with it? Find an example of a challenge you’ve overcome, and explain what you learnt and how you can apply this to future challenges.
+
+Have confidence in yourself and remember that training providers are on your side! We always enjoy reading about your experiences from school, family, jobs and volunteering, and look forward to meeting you at interview where you can demonstrate further how well-informed and interested you are in becoming a TEACHER.
```
Comment (EllieNodder) [inline app/views/content/blog/7-things-to-include-in-your-teacher-training-application.md:60]:
```suggestion
Have confidence in yourself and remember that training providers are on your side! We always enjoy reading about your experiences from school, family, jobs and volunteering. We look forward to meeting you at interview where you can show us how well-informed and interested you are in becoming a TEACHER.
```
Just split up the sentences and changed a bit for readability

**Comment by gemmadallmandfe** [comment]
@emilytraff 

As Gaye said, we have a code freeze at the moment so are not deploying to production unless urgent.

In the meantime, Ellie has suggested a couple of minor changes if you want to have a look - I'll then approve and merge on 4 January.

**Comment by gemmadallmandfe** [comment]
@emilytraff Presume you'd like to get this merged now? If you can just have a quick look at my comment, I'll approve and merge today :-)

**Review by peteryates — APPROVED** [review]
Looks good!

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me!

**Pull request merged**